lib/referencer.js

```diff
@@ -272,7 +272,8 @@ class Referencer extends esrecurse.Visitor {
                 ));
         }
 
-        this.visit(node.superClass);
+		let may_ref_classid = node.superClass && node.superClass.type!=='Identifier'
```

Comment:
```suggestion
		let mayRefClassid = node.superClass && node.superClass.type!=='Identifier'
```

The naming convention for the codebase is camelCase.